draft-ietf-teep-protocol.cddl

```diff
@@ -1,9 +1,9 @@
 teep-message = $teep-message-type .within teep-message-framework
 
 teep-message-framework = [
-  type: uint (0..23) / $teep-type-extension,
+  type: 0..23 / $teep-type-extension,
```

Comment:
Hi @kentakayama,
I prefer `uint (0..23)` rather than the `0..23` to be easier to understand which type is used in cbor.
The cbor uses different types for unsigned integer from 0 to 23 (uses only one byte) and above 23 (uses two bytes).
I wanted to express clearly that we uses unsigned integer from 0 to 23 in the teep.
If the syntax check do not complain about the `uint (0..23)` I would like to keep, but if the cddl tools complaining about it, then I have no objection of changing it to `0..23`.
Other changes in this PR are all fine.

Reply:
This is very late reply, I have talked with @kentakayama over the call and decided to go with `$teep-type = uint .size 1`